packages/vuetify/src/components/VRangeSlider/__tests__/VRangeSlider.spec.ts

```diff
@@ -220,4 +220,12 @@ describe('VRangeSlider.ts', () => {
       expect(wrapper.vm.internalValue).toEqual([0, 0])
     })
   })
+
+  // https://github.com/vuetifyjs/vuetify/issues/6843
+  it('should be uniq id', async () => {
+    const wrapper = mountFunction()
+
+    const [min, max] = wrapper.vm.genInput()
```

Comment:
Move to line 227, keep line 228

Reply:
fixed at https://github.com/vuetifyjs/vuetify/pull/10654/commits/abf6a61de0e52da540c67b388936a38b0f3b771a